node/modules/storageminer.go

```diff
@@ -607,7 +607,7 @@ func StorageProvider(minerAddress dtypes.MinerAddress,
 	df dtypes.StorageDealFilter,
 ) (storagemarket.StorageProvider, error) {
 	net := smnet.NewFromLibp2pHost(h)
-	store, err := piecefilestore.NewLocalFileStore(piecefilestore.OsPath(r.Path()))
+	store, err := piecefilestore.NewLocalFileStore(piecefilestore.OsPath(filepath.Join(r.Path(), "deal-staging")))
```

Comment:
Next steps could be to make this path value configurable via the miner config (I'm sure this would make @travisperson's life a bit easier). If we think it's worth doing I can add it in.